doc/release-notes.md

```diff
@@ -1,20 +1,21 @@
 # release notes
 ## current release
-### release-v3-20190829
-- release date: 2019-08-29
+### release-v4-20190909
+- release date: 2019-09-10
 - status: available
 - changes:
-  - Arriba fusions run with https://github.com/FusionAnnotator/CTAT_HumanFusionLib/wiki#red-herrings-fusion-pairs-that-may-not-be-relevant-to-cancer-and-potential-false-positives to match STAR-fusion results
-  - Clinical data sheet, added information for:
-    - germline_sex_estimate
-    - overall survival (days)
-    - overall survival status (living/deceased)
-    - RNA_library type
-    - edited previous sex variable to be reported_gender
+  - [Clinical V4](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/95):
+    - De-duplicated `BS_6DCSD5Y6` in the clinical file by removing row with wrong age
+    - Added units to `age_at_diagnosis` column (age_at_diagnosis_days)
+  - [new RSEM file](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/93):
+    - Remove QC-failed samples from V3 RSEM. 
+    - Add `BS_XM1AHBDJ` which should be included, but was missing from the V3 RSEM
+  - [new LUMPY file](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/97):
+    - Add data for 628 more subjects which was missing from V3 LUMPY
 - folder structure:
 ```
 data
-└── release-v3-20190829
+└── release-v4-20190909
```

Comment:
same date issue here

Reply:
i know, that's the aws folder name, i created it yesterday but was not clear until today for the RSEM RDS/RDA format discussion. AWS bucket doesn't support "rename", that's why i kept the folder name, but change the release date to 2019-09-10 at line 4.

Reply:
Ahh ok.